examples/tutorials/Exploring_Quantum_Chemistry_with_GDB1k.ipynb

```diff
@@ -224,7 +224,7 @@
     "\n",
     "$\\hat{X_i} = \\frac{X_i - \\mu_i}{\\sigma_i}$\n",
     "\n",
-    "where $\\mu_i$ and $\\sigma_i$ are the mean and standard deviation of the $i$-th feature. This transformation enables the learning to proceed smoothly. A second point is that the atomization energies also fall across a wide range. So we apply an analogous transformation normalization transformation to the output to scale the energies better. We use DeepChem's transformation API to make this happen:"
```

Comment:
why have you removed - `transformation`.

Reply:
I have updated it, please review it again